data/modis.md

```diff
@@ -35,8 +35,8 @@ The following MODIS products are available:
 * [MYD21A2](https://lpdaac.usgs.gov/products/myd21a2v006/) (Aqua Land Surface Temperature/3-Band Emissivity 8-Day L3 Global 1km)
 * [MOD13A1](https://lpdaac.usgs.gov/products/mod13a1v006/) (Terra Vegetation Indices 16-Day L3 Global 500m)
 * [MOD13Q1](https://lpdaac.usgs.gov/products/mod13q1v006/) (Terra Vegetation Indices 16-Day L3 Global 250m)
-* [MYD11A1](https://lpdaac.usgs.gov/products/myd13a1v006/) (Aqua Vegetation Indices 16-Day L3 Global 500m)
-* [MYD11A2](https://lpdaac.usgs.gov/products/myd13q1v006/) (Aqua Vegetation Indices 16-Day L3 Global 250m)
+* [MYD11A1](https://lpdaac.usgs.gov/products/myd11a1v006/) (Aqua Land Surface Temperature/Emissivity Daily L3 Global 1km)
+* [MYD11A2](https://lpdaac.usgs.gov/products/myd11a2v006/) (Aqua Land Surface Temperature/Emissivity 8-Day L3 Global 1km)
```

Comment:
Thanks. I think we also want to link to the myd13a1v006 and myd13q1v006 products.

```suggestion
* [MYD11A1](https://lpdaac.usgs.gov/products/myd13a1v006/) (Aqua Vegetation Indices 16-Day L3 Global 500 m SIN Grid)
* [MYD11A2](https://lpdaac.usgs.gov/products/myd13q1v006/) (Aqua Vegetation Indices 16-Day L3 Global 250 m SIN Grid)
```